app/src/main/java/com/github/se/travelpouch/ui/dashboard/Dashboard.kt

```diff
@@ -0,0 +1,15 @@
+package com.github.se.travelpouch.ui.dashboard
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.setValue
+import com.github.se.travelpouch.model.activity.ActivityViewModel
+import com.github.se.travelpouch.model.dashboard.CalendarViewModel
+import com.github.se.travelpouch.ui.navigation.NavigationActions
+
+@Composable
+fun DashboardScreen(
+    navigationActions: NavigationActions,
+    activityViewModel: ActivityViewModel,
+    calendarViewModel: CalendarViewModel
+) {}
```

Comment:
What's this screen? I'm a bit confused

Reply:
Yes, I forgot to delete it. I will delete the file